docs/source/development.rst

```diff
@@ -25,9 +25,17 @@ libraries. On Debian or Ubuntu, we can install these using
 
     $ sudo apt-get install python-dev zlib1g-dev git libxslt1-dev
 
-.. note::
-    TODO: Document this basic step for other platforms? We definitely want
-    to tell people how to do this with Brew or ports on a Mac.
+Alternatively, on Mac computers, install Anaconda Python
```

Comment:
Doesn't seem like it would make life any easier, as long as we have what we need listed in requirements.txt...

I don't really get the need for anaconda... is the selling point that you load up on packages when you install python and then are less likely to get a 'no package found' error down the line?  If that's the only advantage, we don't need it.